setup.cfg

```diff
@@ -1,9 +1,9 @@
 [metadata]
 name = kintro
-url = https://github.com/brutog/kintro
+url = https://github.com/neckbeard-io/kintro
 project_urls =
-    Source Code = https://github.com/brutog/kintro
-author = bru
```

Comment:
This is the short handle I use when i can actually get it on a website, which is nowhere anymore.